homeassistant/components/samsungtv/config_flow.py

```diff
@@ -170,7 +170,15 @@ async def async_step_import(
         # We need to import even if we cannot validate
         # since the TV may be off at startup
         await self._async_set_name_host_from_input(user_input)
-        self._async_abort_entries_match({CONF_HOST: self._host})
+        entry, _ = self._async_get_existing_matching_entry()
```

Comment:
Why are you not passing in the data to update in the abort entries match?

Reply:
`_abort_if_unique_id_configured` takes updates, but `_async_abort_entries_match` doesn't
I did discover that `async_update_entry` returns a boolean (`changed`) to indicate that an update did take place, so there is no need for me to do that check prior.